src/main/java/org/reaktivity/nukleus/http_cache/internal/stream/HttpCacheProxyCacheableRequest.java

```diff
@@ -516,8 +516,8 @@ private void incAttempts()
             {
                 final String name = h.name().asString();
                 final String value = h.value().asString();
-                if (!CONTENT_LENGTH.equals(name) &&
-                    !AUTHORIZATION.equals(name))
+                if (!CONTENT_LENGTH.equalsIgnoreCase(name) &&
+                    !AUTHORIZATION.equalsIgnoreCase(name))
```

Comment:
We are supposed to be canonicalizing to lower case in `http` and `http2` so that other nuklei don't need to do this ignore case check.

Did you discover an issue with header name case?

Reply:
nope, I forgot about it. Do you want me to revert it back?

Reply:
Yes please.